cobalt/build/configs/BUILD.gn

```diff
@@ -23,10 +23,7 @@ config("cobalt_config") {
     configs += [ "//${starboard_path}/platform_configuration" ]
   }
 
-  defines = [
-    "ENABLE_BUILDFLAG_IS_COBALT",
-    "OS_STARBOARD",
-  ]
+  defines = [ "ENABLE_BUILDFLAG_IS_COBALT" ]
```

Comment:
I know it's unrelated, but would you mind removing this one as well? It's also defined conditionally below

Reply:
Done